test/AdoNetCore.AseClient.Tests/Integration/HealthCheckTests.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace AdoNetCore.AseClient.Tests.Integration
 {
-    [SetUpFixture]
+    //[SetUpFixture]
```

Comment:
@senseibaka  - Is this meant to be checked-in. Why comment it out and not delete it? Or is it a mistake?

Reply:
Woops